uber/templates/regform.html

```diff
@@ -285,10 +285,36 @@
 <div class="form-group">
     <label for="email" class="col-sm-2 control-label">Email Address</label>
     <div class="col-sm-6">
-        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
+        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address" id="mailValue1">
     </div>
 </div>
 
+<div class="form-group">
+    <label for="email" class="col-sm-2 control-label">Email Address Confirmation</label>
+    <div class="col-sm-6">
+        <input type="email" name="email2" id="email2" value=" emailValue2" class="form-control" placeholder="Email Address 2" id="mailValue2">
+    </div>
+</div>
+
+<div hidden class="email-error">
+        <br>
+    <p style="text-indent: 23em; color: red"><b>THE EMAIL Addresses entered do not match, please enter them again.</b></p>
+</div> <br>
+<div hidden class="email-identical">
+        <br>
+    <p style="text-indent: 23em; color: green"><b>THE EMAIL Addresses entered match.</b></p>
+</div> <br>
+
+<script>
+    var emailValue2;
+        if(document.getElementById("mailValue2").value == document.getElementById("mailValue1").value){
+            document.getElementsByClassName("email-identical").removeAttribute("hidden");
+        }
+        else{
+            document.getElementsByClassName("email-error").removeAttribute("hidden");
+        }
+</script>
```

Comment:
Thanks guys, I'll look into JQuery as an option and see what I can do with that. I'm not super strong on JS myself but I'll look into that too. I thought this was all you needed to do to get JS running but I could be very wrong haha. I made some test message windows pop up in that section before so I think a big part of it is getting it to run more than when the page is loaded like you said.